cardano-api/src/Cardano/Api/Orphans.hs

```diff
@@ -90,58 +98,131 @@ instance ToJSON Mary.AssetName where
   toJSON = Aeson.String . Text.decodeLatin1 . B16.encode . Short.fromShort . Mary.assetName
 
 instance ToJSON Shelley.AccountState where
-  toJSON (Shelley.AccountState tr rs) = object [ "treasury" .= tr
-                                               , "reserves" .= rs
-                                               ]
+  toJSON (Shelley.AccountState tr rs) =
+    object
+      [ "treasury" .= tr
+      , "reserves" .= rs
+      ]
+  toEncoding (Shelley.AccountState !tr !rs) =
+    Aeson.pairs $ mconcat
+      [ "treasury" .= tr
+      , "reserves" .= rs
+      ]
```

Comment:
Avoid duplication as mentioned in another comment, also fields are already strict
```suggestion
  toJSON = object . accountStatePairs
  toEncoding = Aeson.pairs . mconcat . accountStatePairs

 accounStatePairs (Shelley.AccountState tr rs) =
      [ "treasury" .= tr
      , "reserves" .= rs
      ]
```